ts41/get.d.ts

```diff
@@ -0,0 +1,82 @@
+import {Split} from '../source/split';
+import {Integers} from '../source/utilities';
+
+/**
+ * Gets a deeply-nested property from an object, like lodash's `get` method.
+ *
+ * Use-case: retrieve a property from deep inside an API response or other complex object.
+ *
+ * @example
+ * import { Get } from 'type-fest'
+ *
+ * interface ApiResponse {
+ *   hits: {
+ *     hits: Array<{
+ *       _id: string
+ *       _source: {
+ *         name: Array<{
+ *           given: string[]
+ *           family: string
+ *         }>
+ *         birthDate: string
+ *       }
+ *     }>
+ *   }
+ * }
+ *
+ * type Name = Get<ApiResponse, 'hits.hits[0]._source.name'> // Array<{ given: string[]; family: string }>
+ *
+ * @explanation
```

Comment:
Is this a valid JSDoc tag? I couldn't find it.

Reply:
No! I was struggling to get formatting to work properly, and didn't mean to leave this in. Removing.